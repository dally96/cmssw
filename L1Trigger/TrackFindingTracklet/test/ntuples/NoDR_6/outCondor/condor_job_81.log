000 (802170.081.000) 02/16 20:57:46 Job submitted from host: <188.185.121.235:9618?addrs=188.185.121.235-9618+[2001-1458-d00-49--100-36e]-9618&alias=bigbird22.cern.ch&noUDP&sock=schedd_8027_6ecd>
...
040 (802170.081.000) 02/16 20:59:12 Started transferring input files
	Transferring to host: <188.184.165.124:9618?addrs=188.184.165.124-9618+[2001-1458-301-51--100-77]-9618&alias=b7s14p9706.cern.ch&noUDP&sock=slot1_21_3842_c955_63846>
...
040 (802170.081.000) 02/16 20:59:12 Finished transferring input files
...
001 (802170.081.000) 02/16 20:59:13 Job executing on host: <188.184.165.124:9618?addrs=188.184.165.124-9618+[2001-1458-301-51--100-77]-9618&alias=b7s14p9706.cern.ch&noUDP&sock=startd_3088_b08b>
...
006 (802170.081.000) 02/16 20:59:22 Image size of job updated: 290476
	104  -  MemoryUsage of job (MB)
	106048  -  ResidentSetSize of job (KB)
...
006 (802170.081.000) 02/16 21:00:52 Image size of job updated: 2957408
	104  -  MemoryUsage of job (MB)
	106152  -  ResidentSetSize of job (KB)
...
040 (802170.081.000) 02/16 21:00:52 Started transferring output files
...
040 (802170.081.000) 02/16 21:00:53 Finished transferring output files
...
005 (802170.081.000) 02/16 21:00:53 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:24, Sys 0 00:00:04  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:24, Sys 0 00:00:04  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	25001  -  Run Bytes Sent By Job
	25479  -  Run Bytes Received By Job
	25001  -  Total Bytes Sent By Job
	25479  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       63       17   3014042 
	   Memory (MB)          :      104     2000      2000 

	Job terminated of its own accord at 2022-02-16T20:00:52Z.
...
