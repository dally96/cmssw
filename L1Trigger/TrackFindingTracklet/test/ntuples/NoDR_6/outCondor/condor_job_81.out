/afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test /pool/condor/dir_78184 
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally/CN=546391038
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/d/dally/private/x509up_u141138
timeleft  : 07:15:13
key usage : Digital Signature, Key Encipherment
=== VO cms extension information ===
VO        : cms
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
issuer    : /DC=ch/DC=cern/OU=computers/CN=cms-auth.web.cern.ch
attribute : /cms/Role=NULL/Capability=NULL
timeleft  : 07:15:12
uri       : voms-cms-auth.app.cern.ch:443

subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally/CN=546391038
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/d/dally/private/x509up_u141138
timeleft  : 07:15:12
key usage : Digital Signature, Key Encipherment
=== VO cms extension information ===
VO        : cms
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
issuer    : /DC=ch/DC=cern/OU=computers/CN=cms-auth.web.cern.ch
attribute : /cms/Role=NULL/Capability=NULL
timeleft  : 07:15:11
uri       : voms-cms-auth.app.cern.ch:443

cmsRun /afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test/customise_L1TrackNtupleMaker.py inputFiles_clear inputFiles_load=/afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test/ntuples/NoDR_6/pFiles_81.txt outputFile=outputL1Ntuple_81.root print
Singletons:
  filePrepend        : 
                     - String to prepend location of all files
  maxEvents          : -1
                     - Number of events to process (-1 for all)
  outputFile         : outputL1Ntuple_81.root
                     - Name of output file (if needed)
  secondaryOutputFile: 
                     - Name of second output file (if needed)
  section            : 0
                     - This section (from 1..totalSections inclusive)
  tag                : 
                     - tag to add to output filename
  totalSections      : 0
                     - Total number of sections
Lists:
  inputFiles         : ['/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/b3c8ad5e-c167-4e42-a6e9-a1a821049ee8.root' ]
                     - Files to process
  secondaryInputFiles: []
                     - Second group of files to process (if needed)

using geometry D76 (tilted)
Wed Feb 16 21:00:38 CET 2022
Thread 7 (Thread 0x2b4dd436c700 (LWP 264)):
#0  0x00002b4d7afeab3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b4d7afeabcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b4d7afeac6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b4d81ad6816 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b4d81ad68c9 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#5  0x00002b4d7afe4ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b4d7b2f7b0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2b4dd416b700 (LWP 263)):
#0  0x00002b4d7afeab3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b4d7afeabcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b4d7afeac6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b4d81ad6816 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b4d81ad68c9 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#5  0x00002b4d7afe4ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b4d7b2f7b0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2b4dd3f6a700 (LWP 262)):
#0  0x00002b4d7afeab3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b4d7afeabcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b4d7afeac6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b4d81ad6816 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b4d81ad68c9 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#5  0x00002b4d7afe4ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b4d7b2f7b0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2b4dd3d69700 (LWP 261)):
#0  0x00002b4d7afebe9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002b4d809436ff in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#2  0x00002b4d81a78b0a in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#3  0x00002b4d81a78c69 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b4d7afe4ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b4d7b2f7b0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2b4dd3b68700 (LWP 260)):
#0  0x00002b4d7b2f80e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002b4d8093ded2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#2  0x00002b4d8093a78d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#3  0x00002b4d80942f68 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#4  0x00002b4d7afe4ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b4d7b2f7b0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2b4d9a11c700 (LWP 253)):
#0  0x00002b4d7afec1d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002b4d81d39387 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#2  0x00002b4d81d3a01a in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#3  0x00002b4d7abb5af0 in std::execute_native_thread_routine (__p=0x2b4d992f3800) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002b4d7afe4ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b4d7b2f7b0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2b4d7d373180 (LWP 235)):
#0  0x00002b4d7b2ecddd in poll () from /lib64/libc.so.6
#1  0x00002b4d81d397b7 in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#2  0x00002b4d81d3a0ec in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#3  0x00002b4d81d3d59b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002b4d7b22f387 in raise () from /lib64/libc.so.6
#6  0x00002b4d7b230a78 in abort () from /lib64/libc.so.6
#7  0x00002b4d7ab80683 in __gnu_cxx::__verbose_terminate_handler () at ../../../../libstdc++-v3/libsupc++/vterminate.cc:95
#8  0x00002b4d7ab8c0a6 in __cxxabiv1::__terminate (handler=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:48
#9  0x00002b4d7ab8b1a9 in __cxa_call_terminate (ue_header=ue_header@entry=0x2b4de3752d60) at ../../../../libstdc++-v3/libsupc++/eh_call.cc:54
#10 0x00002b4d7ab8bad4 in __cxxabiv1::__gxx_personality_v0 (version=<optimized out>, actions=6, exception_class=5138137972254386944, ue_header=0x2b4de3752d60, context=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_personality.cc:677
#11 0x00002b4d7afd27b3 in _Unwind_RaiseException_Phase2 (exc=0x2b4de3752d60, context=0x7ffe53cf2530, frames_p=0x7ffe53cf2438) at ../../../libgcc/unwind.inc:64
#12 0x00002b4d7afd3016 in _Unwind_Resume (exc=0x2b4de3752d60) at ../../../libgcc/unwind.inc:241
#13 0x00002b4d7933ab1c in TFile::Close(char const*) [clone .cold] () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libRIO.so
#14 0x00002b4d80c388ee in TStorageFactoryFile::~TStorageFactoryFile() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolTFileAdaptor.so
#15 0x00002b4d80c38919 in TStorageFactoryFile::~TStorageFactoryFile() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolTFileAdaptor.so
#16 0x000000000040f4d8 in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() ()
#17 0x00002b4de7061b6c in edm::PoolOutputModule::~PoolOutputModule() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolOutput.so
#18 0x00002b4de7061daf in edm::PoolOutputModule::~PoolOutputModule() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolOutput.so
#19 0x00002b4dd04b7c03 in std::_Sp_counted_deleter<edm::one::OutputModuleBase*, std::default_delete<edm::one::OutputModuleBase>, std::allocator<void>, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreModules.so
#20 0x00002b4dd04b8132 in std::_Sp_counted_ptr_inplace<edm::maker::ModuleHolderT<edm::one::OutputModuleBase>, std::allocator<edm::maker::ModuleHolderT<edm::one::OutputModuleBase> >, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreModules.so
#21 0x00002b4d78a40d7c in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#22 0x00002b4d78a40d06 in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#23 0x00002b4d78a40d06 in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#24 0x00002b4d78a40dd6 in std::_Sp_counted_ptr<edm::ModuleRegistry*, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#25 0x00002b4d78957598 in std::default_delete<edm::Schedule>::operator()(edm::Schedule*) const [clone .isra.0] [clone .part.0] () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#26 0x00002b4d7895ebe2 in edm::EventProcessor::~EventProcessor() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#27 0x000000000040b851 in (anonymous namespace)::EventProcessorWithSentry::~EventProcessorWithSentry() ()
#28 0x000000000040b649 in main ()
